packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -218,9 +218,15 @@ export default class WorkspacesForeachCommand extends BaseCommand {
 
     let abortNextCommands = false;
 
+    // get the configuration again but override messageNames so that we don't double-nest message names
+    // in the foreach output stream if they're enabled.
+    const configWithoutMessageNames = await Configuration.find(this.context.cwd, this.context.plugins);
+    configWithoutMessageNames.values.set(`enableMessageNames`, false);
```

Comment:
This doesn't seem ideal, the `StreamReport` could instead have a `includePrefix` option that disabled both the "caret" and the prefix.

Reply:
Agree that this feels hacky and that feels cleaner, I'll go ahead and update the PR.